woocommerce/woocommerce-android
# Remove in app feedback feature flag

Closes #2766

This enables the In-app Feedback feature for all users.

## Testing

1. Delete the app from the device.
2. Run and log in.
3. Navigate to Products. 
4. Confirm that the banner and the “Give Fedback” and ”Dismiss” buttons are visible.
5. Time-travel to 3 months from now. 
6. Navigate to My Store. 
7. Confirm that the card with buttons “Could Be Better” and “I Like It” is visible. 

Update release notes:
